bin/metaconfig

```diff
@@ -1,11 +1,10 @@
-#!/pro/bin/perl
+#!/usr/bin/perl
 
-chdir "/pro/3gl/CPAN/perl";
-system "chown merijn Configure config_h.SH";
-chmod 0775, "Configure", "config_h.SH";
```

Comment:
This is probably still needed and very depending on what your devel environment is. If these files are read-only your process will die is weird ways. I'm OK with a die if either is read-only